src/main.ts

```diff
@@ -0,0 +1,25 @@
+import { fetchProjectItems } from "./fetchProjectItems";
+import { config } from "./config";
+import * as fs from "fs";
+
+const OUTPUT_FILE = `${process.cwd()}/dist/roadmap.json`
+
+const { projectId, org } = config;
+
+(async () => {
+  try {
+    const project = await fetchProjectItems({ org, projectId, first: 100 })
+    console.log(project)
+    try {
+      await fs.mkdir('./dist', () => {
+        fs.writeFileSync(OUTPUT_FILE, JSON.stringify(project), )
+        console.log(`Created ${OUTPUT_FILE} 👍`)
+      })
+    } catch (e) {
+      console.error(e)
+    }
+  } catch (e) {
+    console.error(e)
+  }
+
+})().catch(console.error)
```

Comment:
What's the reason behind all these catches? Why do we need the code not to do anything, yet not fail?

Reply:
@mutantcornholio it was failing silently without any logs and 0 result, so i added these, but I remember during iterations of refactoring there was a mistake on how it was handled, so may be that was a reason.
So you propose just remove all try/catches and let it fail with errors in console? that sounds right to me

Reply:
Considering that you're handling async code, I propose something like this:
```ts
(async () => {
    const project = await fetchProjectItems({ org, projectId, first: 100 })
    await fs.mkdir("./dist")
    await fs.writeFile(OUTPUT_FILE, JSON.stringify(project))
    console.log(`Created ${OUTPUT_FILE} 👍`)
})().catch(e => {
    console.error(e);
    process.exit(1);
})
```

Reply:
[1970a31](https://github.com/paritytech/roadmap/pull/10/commits/1970a311259a87f2f52d8d330443b1fa13c289fe)